services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -323,21 +457,28 @@ private string GetFmsData(string story, string key, IngestModel ingest)
                 // Iterate over the list of stories and add a new item to the articles list for each story.
                 foreach (XmlElement story in elementList)
                 {
-                    var item = new SourceContent();
                     var papername = GetXmlData(story, Fields.Papername, ingest);
-                    item.Source = GetItemSourceCode(ingest, papername);
-                    item.Title = GetXmlData(story, Fields.Headline, ingest);
-                    item.Uid = GetXmlData(story, Fields.Id, ingest);
-                    item.Body = GetXmlData(story, Fields.Story, ingest);
-                    item.FilePath = document.Key;
-                    item.Summary = GetXmlData(story, Fields.Summary, ingest);
-                    item.Page = GetXmlData(story, Fields.Page, ingest);
-                    item.Section = GetXmlData(story, Fields.Section, ingest);
-
-                    item.Authors = GetAuthorList(GetXmlData(story, Fields.Author, ingest));
-                    item.PublishedOn = GetPublishedOn(GetXmlData(story, Fields.Date, ingest), ingest);
-
-                    articles.Add(item);
+                    var code = GetItemSourceCode(ingest, papername, sources);
+
+                    if (!string.IsNullOrEmpty(code)) // This is a valid newspaper source
```

Comment:
If there is no code we seem to ignore it.  Shouldn't we at least log it?

Reply:
A warning is logged in GetItemSourceCode().